### PR TITLE
EDSC-2545: Sidebar and collection results styling

### PR DESCRIPTION
**Pull request by trevorlang**

### CODE REVIEW

File: static/src/js/components/CollectionResults/CollectionResultsItem.scss [hunk 1]
```diff
@@ -14,17 +14,17 @@
     }
   }
 
-  &__body {
-    width: 100%;
-  }
+  // &__body {
```
Comment (abbottry) [inline static/src/js/components/CollectionResults/CollectionResultsItem.scss]: delete?

File: static/src/js/components/CollectionResults/CollectionResultsItem.scss [hunk 2]
```diff
@@ -14,17 +14,17 @@
...
+  // }
 
   &__body-primary {
     display: flex;
   }
 
-  &__info {
-    width: 100%;
-  }
+  // &__info {
```
Comment (abbottry) [inline static/src/js/components/CollectionResults/CollectionResultsItem.scss]: delete?

File: static/src/js/components/FilterStack/FilterStackItem.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@
   flex-direction: row;
   align-items: center;
   flex-wrap: wrap;
-  padding: $spacer/2 $spacer*0.78 $spacer/2 $spacer;
+  padding: $spacer/2 $spacer*0.78 $spacer/2 $spacer*0.925;
```
Comment (abbottry) [inline static/src/js/components/FilterStack/FilterStackItem.scss]: These are very precise values, where did you come up with these?

File: static/src/js/components/FilterStack/FilterStackItem.scss [hunk 4]
```diff
@@ -16,13 +16,13 @@
...
     justify-content: flex-start;
-    width: 1.5rem;
+    margin-bottom: 0.125rem;
   }
 
   &__icon {
     text-align: center;
     color: $color__black--400;
-    font-size: 1.25rem;
+    font-size: .925rem!important;
```
Comment (abbottry) [inline static/src/js/components/FilterStack/FilterStackItem.scss]: Does this still work without the space between the value the `!important`?

File: static/src/css/utils/variables/_shadows.scss [hunk 5]
```diff
@@ -1,2 +1,3 @@
 $shadow__default: 0 0 7px rgba(0,0,0,0.1);
-$shadow__panel: 0 6px 12px rgba(0,0,0,0.175);
\ No newline at end of file
+$shadow__panel: 0 6px 12px rgba(0,0,0,0.175);
+$shadow__panel-dark: 0 6px 12px rgba(0,0,0,0.5);
```
Comment (trevorlang) [inline static/src/css/utils/variables/_shadows.scss]: Extra line

File: static/src/js/components/CollectionResults/CollectionResultsHeader.scss [hunk 6]
```diff
@@ -23,8 +28,17 @@
...
     }
   }
+
+  &__meta {
+    padding: 0.25rem 1rem;
+    border-top: 1px solid $color__black--200;
+    font-size: 0.75rem;
+    color: $color__black--600;
+  }
 }
```
Comment (trevorlang) [inline static/src/js/components/CollectionResults/CollectionResultsHeader.scss]: Extra line

File: static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.scss [hunk 7]
```diff
@@ -37,7 +41,7 @@
     color: $color__black--500;
 
     .dropdown-item:hover & {
-      color: inherit;
+      color: $color__black--500;
     }
   }
 }
```
Comment (trevorlang) [inline static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.scss]: Extra line

**Pull request merged**